src/mds/mdstypes.h

```diff
@@ -45,9 +45,10 @@
 
 // No longer created but recognised in existing filesystems
 // so that we don't try to fragment it.
-#define MDS_INO_CEPH              2
+#define MDS_INO_CEPH              2 /* hidden .ceph dir */
 
 #define MDS_INO_GLOBAL_SNAPREALM  3
+#define MDS_INO_LOST_AND_FOUND    4 /* reserved ino for use in recovery */
```

Comment:
Hmm, these definitions are in `src/include/ceph_fs.h` as CEPH_INO_CEPH and CEPH_INO_LOST_AND_FOUND.

I think we ought to add a definition for CEPH_INO_GLOBAL_SNAPREALM and get rid of MDS_INO_CEPH and MDS_INO_GLOBAL_SNAPREALM.

@lxbsz Would you mind doing that in the context of this patch? Having this defined in two places seems wrong.

Reply:
Sure, make sense to me. Will fix it.